_episodes/07-raster-calculations.md

```diff
@@ -3,303 +3,367 @@ title: "Raster Calculations in Python"
 teaching: 40
```

Comment:
Checked together and this seems to be browser-dependent (on OS X it seems to work on Safari but not on Chrome). Since equations are rather simple, agreed to leave it as it is